### PR TITLE
[qa]: enable rpcbind_test

### PR DESCRIPTION
**Pull request by rodentrabies**
Added parsing for `'<host>:<port>'` arguments to `rpc_url()`, as it was required for this test to work. Also, changed username and password that were passed to `get_rpc_proxy()` in `run_allowip_test()` to hardcoded (default) ones via call to `rpc_url()`.

### CODE REVIEW

**Comment by laanwj** [comment]
Thanks, utACK https://github.com/bitcoin/bitcoin/pull/8400/commits/9bbb414b800cf71f93f15d2cb5d4cf32c72294e6

**Comment by yurizhykin** [comment]
For #8351

**Comment by MarcoFalke** [comment]
Thank you for fixing the issue!

Concept ACK. Will take a closer look in the coming days...

File: qa/rpc-tests/rpcbind_test.py [hunk 1]
```diff
@@ -5,143 +5,108 @@
 
 # Test for -rpcbind, as well as -rpcallowip and -rpcconnect
 
-# TODO extend this test from the test framework (like all other tests)
-
 import tempfile
 import traceback
```
Comment (MarcoFalke) [inline qa/rpc-tests/rpcbind_test.py]: Nit: You can remove the imports as well.

**Pull request merged**